.gitignore

```diff
@@ -10,6 +10,9 @@ generic.key
 # git secret
 .gitsecret/keys/random_seed
 
+# VCR cassettes
+test/vcr_cassettes/
```

Comment:
Why do we ignore these? Aren't they necessary to make sure we're all running the same tests?